pkg/fleet/env/env.go

```diff
@@ -31,6 +31,7 @@ const (
 	envAgentMajorVersion     = "DD_AGENT_MAJOR_VERSION"
 	envAgentMinorVersion     = "DD_AGENT_MINOR_VERSION"
 	envApmLanguages          = "DD_APM_INSTRUMENTATION_LANGUAGES"
+	envCDNLocalDirPath       = "DD_INSTALLER_CDN_LOCAL_DIR_PATH"
```

Comment:
nit: maybe add a _DEBUG or something to really avoid suggesting this should be used :D